webapp/src/containers/Rhs/index.tsx

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 
 import usePluginApi from 'hooks/usePluginApi';
-import {getProjectDetailsState} from 'selectors';
+
+import {getProjectDetailsState, getWebsocketEventState} from 'selectors';
 
 import AccountNotLinked from './accountNotLinked';
 import ProjectList from './projectList';
 import ProjectDetails from './projectDetails';
 
 const Rhs = (): JSX.Element => {
-    const {isUserAccountConnected, state} = usePluginApi();
+    const {state} = usePluginApi();
+    const {isConnected} = getWebsocketEventState(state);
 
     return (
         <div className='overflow-auto height-rhs position-relative padding-16'>
             {
-                !isUserAccountConnected() && <AccountNotLinked/>
+                !isConnected && <AccountNotLinked/>
             }
             {
-                isUserAccountConnected() && (
+                isConnected && (
```

Comment:
```suggestion
            {!isConnected ? <AccountNotLinked/> : (
```

Reply:
if we do this 
```
{
                isConnected ? (
                    getProjectDetailsState(state).projectID ?
                        <ProjectDetails {...getProjectDetailsState(state)}/> :
                        <ProjectList/>
                ) : <AccountNotLinked/>
            }
```

we would be having a nested ternary operator which is not allowed by MM webapp eslint